dependencies.props

```diff
@@ -47,17 +50,18 @@
     
     <XmlUpdateStep Include="CoreFx">
       <Path>$(MSBuildThisFileFullPath)</Path>
-      <ElementName>PlatformPackageVersion</ElementName>
-      <PackageId>Microsoft.NETCore.Platforms</PackageId>
+      <ElementName>CoreFxExpectedPrerelease</ElementName>
```

Comment:
Instead, just have two separate properties which get updated appropriately:

`<CoreFxVersion>4.4.0-preview2-25310-01</CoreFxVersion>`
`<PlatformPackageVersion>2.0.0-preview2-25310-01</PlatformPackageVersion>`

These two things (especially in servicing releases) can and will have different "pre-release" versions.  Say when we just change the CoreFx package, but keep the PlatformPackage unchanged.